### DISCUSSION TITLE
vue.js

### DISCUSSION THREAD

**Discussion by bardiotmarin** [simeydotme/pokemon-cards-css#24] [General]
Hey buddy , i search to implement on vue.js project did you have some idea to do this more easy ?

**Comment by simeydotme** [comment]
the html and css are able to be copied and viewed from the demo site,
as for the js and interactions ... sorry you're on your own there. The UX is pretty complicated and specific to the implementation. 

But my "v1" pokemon card demo on CodePen has a very simplified and basic interaction which you should be able to understand how it works :)

**Reply by bardiotmarin** [reply]
Thanks for the reply buddy :)

**Comment by Maurier** [comment]
@bardiotmarin Here is a simple Vue implementation: https://github.com/Maurier/vue-pokemon-cards-css

**Reply by simeydotme** [reply]
hehe nice :)
It's amazing how similar the code is after Vue3 Composition API. 
Shame Vue's Templates are still a bit too obtuse and also still uses a VirtualDOM otherwise it'd be a strong contender for Svelte! :D 

Hope you learned something, and found it fun! :)

**Reply by Maurier** [reply]
Thanks a lot for sharing! This made my weekend, I learned so much.

For this implementation I stayed with Vue 2 and only pulled in the composition api because of a dependency.

I am not a fan of the composition api yet, and prefer to use Vue 2 with dev dependencies like vue-class-components because I come from a C++ background.

**Reply by simeydotme** [reply]
well it looks clean! 

I know my original code is a bit chaotic and not well architected as it's just a tech demo, so you're working from shaky foundations haha.

I totally didn't know that Vue has the spring tool until I read your code :D

**Reply by bardiotmarin** [reply]
@Maurier Very helpfull to understand the mechanic thanks a lot !

**Reply by zzz6519003** [reply]
react one?
